src/te/schedule/schedule_dataflow_rewrite.cc

```diff
@@ -614,10 +614,74 @@ void InjectInline(ScheduleNode* sch) {
   }
 }
 
+void LegalizeInvalidAttach(ScheduleNode* sch) {
```

Comment:
```suggestion
void LegalizeInvalidAttach(ScheduleNode* sch) {
  // This legalizes the compute_at location if the target iterator of compute_at was split or fused.
  // Case 1: If the target of compute_at is split, we will move the compute_at location to the inner iterator.
  // Case 2: If the target of compute_at is fused, we will move the compute_at location to the newly fused iterator.
  // Note that case 2 can only happen if the target of compute_at is the innermost iterator of fuse operation.
```

Document this function